webviz_config/webviz_store.py

```diff
@@ -167,7 +173,10 @@ def build_store(self):
                 else:
                     raise ValueError(f"Unknown return type {type(output)}")
 
-                print(" \033[92m\033[1m[\u2713] Saved\033[0m")
+                counter += 1
```

Comment:
Not sure if it is better but could get counter by
`for counter, argtuples in enumerate(self.storage_function_argvalues[func]):`

Reply:
That would have been better, but there is an outer loop as well which prevents tha (outer is on the different functions to call, inner on the different arg combinations to a given function).